onyo/lib/onyo.py

```diff
@@ -637,29 +638,22 @@ def get_asset_content(self,
         if not self.is_asset_path(path):
             raise NotAnAssetError(f"{path} is not an asset path")
         try:
-            if self.is_inventory_dir(path):
-                # It's an asset and an inventory dir -> asset dir
-                a = get_asset_content(path / self.ASSET_DIR_FILE_NAME)
-                a['is_asset_directory'] = True
-            else:
-                a = get_asset_content(path)
-                a['is_asset_directory'] = False
+            ## Where do we make sure to distinguish onyo.path.file from onyo.path.relative?
```

Comment:
This sounds like a TODO. Can it be marked as such? Otherwise we will need to read all comments to make sure they don't contain unmarked TODOs.

Reply:
Agree. Will do.